gr-utils/python/modtool/scm.py

```diff
@@ -81,6 +81,13 @@ def is_active(self):
         """ Returns true if this repository manager is operating on an active, source-controlled directory. """
         return self.is_empty
 
+    def get_gituser(self):
+        """ Gets the git user """
+        try:
+            return subprocess.check_output('git config --global user.name', shell=True).strip()
+        except (OSError, subprocess.CalledProcessError):
+            return 0
```

Comment:
My only complaint: this should be `None` instead of `0`. `0` wouldn't be a string (which is what we're expecting a git user to be), but still semantically different from `None`, even if both evaluate to `False` in an `if` clause :) Can you please replace 0 by `None`?